build.gradle

```diff
@@ -0,0 +1,120 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group "org.logstash.outputs"
```

Comment:
Just to understand, now instead of building the gem via the gemfile, we have a gradle project?

Reply:
That is right! dependencies come in from gradle and the packaging of these dependencies come from gem